R/read_habitatdata.R

```diff
@@ -40,6 +40,17 @@
 #'   each of them. The variable \code{certain} will be \code{FALSE} if
```

Comment:
About issue [#117](https://github.com/inbo/n2khab/issues/117): 

> you may want to add a note on the interpretation of phab for code combinations from one column of the original habitatmap

I was thinking about describing this behaviour (for 31xx_rbbmr as well as uncertain types) in the geospatial hab [vignette](https://inbo.github.io/n2khab/articles/v025_geospatial_hab.html), but I changed my mind: I think this can be useful here too, so:

```suggestion
#'   each of them, with \code{phab} for each new row simply set to the original 
#'   value of \code{phab}. 
#'   The variable \code{certain} will be \code{FALSE} if
```

Reply:
Well seen - I'dmissed the exact meaning of that last bullet.